pkg/virt-api/rest/dialers.go

```diff
@@ -0,0 +1,72 @@
+package rest
+
+import (
+	"fmt"
+	"net"
+
+	restful "github.com/emicklei/go-restful"
+	"k8s.io/apimachinery/pkg/api/errors"
+
+	v1 "kubevirt.io/client-go/api/v1"
+	"kubevirt.io/client-go/kubecli"
+	"kubevirt.io/client-go/log"
+)
+
+func netDialer(request *restful.Request) dialer {
+	return func(vmi *v1.VirtualMachineInstance) (net.Conn, *errors.StatusError) {
+		logger := log.Log.Object(vmi)
+
+		targetIP, err := getTargetInterfaceIP(vmi)
+		if err != nil {
+			logger.Reason(err).Error("Can't establish TCP tunnel.")
+			return nil, errors.NewBadRequest(err.Error())
+		}
+
+		port := "22"
```

Comment:
Shouldn't `port` be a required parameter?

Reply:
Done